src/gui/gui.pri

```diff
@@ -36,14 +36,14 @@ HEADERS += \
     $$PWD/rss/htmlbrowser.h \
     $$PWD/rss/rsswidget.h \
     $$PWD/scanfoldersdelegate.h \
-    $$PWD/search/pluginselectdlg.h \
-    $$PWD/search/pluginsourcedlg.h \
+    $$PWD/search/pluginselectdialog.h \
+    $$PWD/search/pluginsourcedialog.h \
     $$PWD/search/searchlistdelegate.h \
     $$PWD/search/searchsortmodel.h \
-    $$PWD/search/searchtab.h \
+    $$PWD/search/searchtabwidget.h \
```

Comment:
This name is confusing (seems like it inherits QTabWidget). It would be better to rename this widget according to its meaning. Since it is interface with the one of search job, it should be SearchJobWidget.

Reply:
WIll change.

Reply:
Should i also change every mention to searchtab or tab to job?
 e.g m_currentSearchTab -> m_currentSearchJob

Reply:
>Should i also change every mention to searchtab or tab to job?

No. It's used as tab of QTabWidget.

Reply:
Ok.